plonky/src/field_gl.rs

```diff
@@ -433,6 +433,14 @@ impl crate::ff::Field for Fr {
         self.mul_assign(&self.clone());
     }
 }
+
+impl Fr {
+    #[inline]
+    pub fn exp(self, power: PositiveInteger) -> Fr {
+        exp(self, power)
+    }
+}
+
 type PositiveInteger = u64;
 #[inline(always)]
 pub fn exp(base: Fr, power: PositiveInteger) -> Fr {
```

Comment:
Unify the `Fr::exp` and `fn exp`  into one and as `a method of Fr` , will be better.